### ISSUE TITLE
VideoTags saves same value for all the videos

### ISSUE THREAD

**Issue by leonidaswilliams** [WWBN/AVideo#3255]
Hi!

I have activated the VideoTags Plugin.

After activate it I have installed the tables.

Then I create two Tag Type.

I go for Video 1 and in the Tag I write 'uno'.

I go for Video 2 and in the Tag (i see 'uno' value) and write 'dos'

I go for Video 1 and the value for the Tag is 'dos'

Can you help me please?

Thanks in advance

**Comment by DanielnetoDotCom** [comment]
Hi

sorry for the late reply

I tried to reproduce this but and I could not. the tags works properly on my side

**Comment by leonidaswilliams** [comment]
Hi

I investigated more and my problem is that in the POST the values for the Video Tags are not sent. I looked the HTML and I've found that when the edition modal is load I have an jquery error:

![image](https://user-images.githubusercontent.com/22470075/86331939-0ffc0d80-bc4a-11ea-8961-0d2f186e28e1.png)

Maybe is stopping the javascript and this is why the values are not sent? I've looked that videoTags are managed with JavaScript.

Thanks
